setup.py

```diff
@@ -28,12 +28,13 @@
     "pandas >=0.24.2",
     "pyarrow >=3.0.0, <7.0dev",
     "pydata-google-auth",
-    "google-api-core >=1.14.0",
+    "google-api-core >=1.21.0",
```

Comment:
Seems unrelated, and unmotivated by anything in the [changelog for that release](https://github.com/googleapis/python-api-core/releases/tag/v1.21.0).

Reply:
Correct. Needed to update due to updating the minimum `google-cloud-bigquery`, though.

We do use `google-api-core` directly, so I think it makes sense to include here still.